frontend_lib/src/component/PopinFixed/PopinFixed.styl

```diff
@@ -18,12 +18,14 @@
     align-items center
     padding 15px 10px
     height 62px
+    & > *
```

Comment:
Why a star? Would it be possible to use a specific selector?

Reply:
Because I want that wall elements have this margin as default.

Reply:
All right. I'm not sure using a star is the best way to do this, instead of specifying the specific tags that we know will be in the header (which is something we control), but I may be missing something here so feel free to resolve the conversation.